presto-raptor/src/main/java/com/facebook/presto/raptor/storage/StorageManagerConfig.java

```diff
@@ -59,6 +61,7 @@
     private DataSize maxShardSize = new DataSize(256, MEGABYTE);
     private DataSize maxBufferSize = new DataSize(256, MEGABYTE);
     private int oneSplitPerBucketThreshold;
+    private String shardDayBoundaryTimeZone = TimeZone.getDefault().getID();
```

Comment:
I think it should be explicitly set to UTC by default to match existing behavior.